### PR TITLE
ipykernel: improves code coverage

### PR DESCRIPTION
**Pull request by jesslatimer**
Changes made:

- Made buffers more variable
- Change input of `deserialized_obj` to be more appropriate
- Fixed a spelling mistake of the variable `deserialized_obj`

### CODE REVIEW

File: projects/ipykernel/fuzz_unpack_roundtrip.py [hunk 1]
```diff
@@ -18,20 +18,26 @@
...
+  max_range = fdp.ConsumeIntInRange(3, 50)
+  for _ in range(3, max_range):
+    buffers.append(fdp.ConsumeBytes(fdp.ConsumeIntInRange(0, 200)))
+  return buffers
+
+
 def TestOneInput(data):
-  if len(data) < 48:
-    return
+  if len(data) < 48:
```
Comment (DavidKorczynski) [inline projects/ipykernel/fuzz_unpack_roundtrip.py:31]: nit: whitespace

File: projects/ipykernel/fuzz_unpack_roundtrip.py [hunk 2]
```diff
@@ -18,20 +18,26 @@
...
-  ]
+
   try:
+    buffers = ConsumeRandomLengthBufferList(fdp)
     f, args, kwargs = unpack_apply_message(buffers)
-  except Exception as e:
+  except Exception:
     return
-
+
```
Comment (DavidKorczynski) [inline projects/ipykernel/fuzz_unpack_roundtrip.py:40]: nit: whitespace

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]
LGTM with minor nits

File: projects/ipykernel/fuzz_unpack_roundtrip.py (hunk 2 above)
Comment (jesslatimer) [reply projects/ipykernel/fuzz_unpack_roundtrip.py]: Done

File: projects/ipykernel/fuzz_unpack_roundtrip.py (hunk 1 above)
Comment (jesslatimer) [reply projects/ipykernel/fuzz_unpack_roundtrip.py]: Done

File: projects/ipykernel/fuzz_unpack_roundtrip.py [hunk 3]
```diff
@@ -18,29 +18,35 @@
...
+    pack_apply_message(f, args, kwargs)
 
 
 def main():
-  atheris.instrument_all()
-  atheris.Setup(sys.argv, TestOneInput)
-  atheris.Fuzz()
+    atheris.instrument_all()
+    atheris.Setup(sys.argv, TestOneInput)
+    atheris.Fuzz()
```
Comment (DavidKorczynski) [inline projects/ipykernel/fuzz_unpack_roundtrip.py:48]: Could we keep the two spaces as indentation in this file?

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]

File: projects/ipykernel/fuzz_unpack_roundtrip.py (hunk 3 above)
Comment (jesslatimer) [reply projects/ipykernel/fuzz_unpack_roundtrip.py]: Done

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**